nni/nas/strategy/evolution.py

```diff
@@ -167,3 +167,36 @@ def _remove_failed_models_from_running_list(self):
             self._running_models = [g for g in self._running_models if g[1].status != ModelStatus.Failed]
             if number_of_failed_models > 0:
                 _logger.info('%d failed models are ignored. Will retry.', number_of_failed_models)
+
+
+class RandomEvolution(RegularizedEvolution):
```

Comment:
What is the difference between this and its super-class?

Reply:
This is a Random method without Evolution, the performance of random sampled subnet is also a strong baseline method.

Reply:
There is a strategy.Random(). Might be what you need.